docs/github-connector/README.md

```diff
@@ -0,0 +1,15 @@
+# GitHub Connector
+
+## Overview
+
+The GitHub Connector is an additional tool for handling GitHub events in Kyma. It registers GitHub API and events in Kyma Application Registry. Then, it converts events incoming from connected GitHub webhooks into format acceptable by Kyma Event Bus and forwards them. For now it handles those events:
+
+- new pull request,
+- new issue,
+- new review on pull request.
```

Comment:
```suggestion
- new pull request review.
```

As per [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md):
> Avoid ending bullet points with semicolons or commas.

Please modify all the bullet points.